transformer/transformers/ampruntimejs_test.go

```diff
@@ -0,0 +1,70 @@
+package transformers_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ampproject/amppackager/transformer/internal/amphtml"
+	tt "github.com/ampproject/amppackager/transformer/internal/testing"
+	"github.com/ampproject/amppackager/transformer/transformers"
+	"golang.org/x/net/html"
+)
+
+func TestAmpRuntimeJS(t *testing.T) {
+	tcs := []tt.TestCase{
+		{
+			Desc:     "no script node",
+			Input:    "<head></head><body></body>",
+			Expected: "<head></head><body></body>",
+		},
+		{
+			Desc:     "no prefix",
+			Input:    "<script src=main.js/>",
+			Expected: "<script src=main.js/>",
+		},
+		{
+			Desc:     "no suffix",
+			Input:    "<script src=\"https://cdn.ampproject.org\"/>",

```

Comment:
Nit: We can use ``` ` ``` tick strings, so we can avoid the `\"` escaping.
```suggestion
			Input:    `<script src="https://cdn.ampproject.org"/>`,

```

Reply:
Done.